src/corehost/cli/fxr/fx_muxer.cpp

```diff
@@ -329,8 +330,20 @@ int fx_muxer_t::execute(const int argc, const pal::char_t* argv[])
             else
             {
                 trace::verbose(_X("Executing as a standalone app as per config file [%s]"), config_file.c_str());
-                corehost_init_t init(deps_file, probe_path, _X(""), host_mode_t::muxer, &config);
                 pal::string_t impl_dir = get_directory(app_or_deps);
+                if (!library_exists_in_dir(impl_dir, LIBHOSTPOLICY_NAME, nullptr) && !probe_path.empty() && !deps_file.empty())
```

Comment:
Yes, this is only for `dotnet build` which doesn't copy the hostpolicy into the bin directory. I was under the same impression, until @piotrpMSFT clarified this. The idea was, as per @piotrpMSFT, we'd be given a `deps file` and a `.dll` and a `additionalprobingpath` to resolve. Earlier this worked because corehost was copied to bin. Now, with muxer based activation of `dotnet build` there is no hostpolicy.dll which should be resolved against the standard meta package.